yaml/boundaries.yaml

```diff
@@ -153,6 +153,25 @@ filters:
       claimed_by: {col: claimed_by}
       recognized_by: {col: recognized_by}
     table: osm
+  # these filters look for claims that don't have a claimed_by
```

Comment:
Expand the comment to explain in 2nd line:

> # but have different point of view tags for admin_level:XX

Reply:
Up above this section we have inline documentation like:

``` 
  # we track two kinds of features from OSM for indicating disputes:
  #
  # 1. **claims**, which have `boundary=claim, claimed_by=*, admin_level=*`
  #    tags. these indicate borders which aren't generally recognized, but
  #    should appear in the viewpoint of the `claimed_by=` value. additionally,
  #    extra viewpoints which don't make a claim for themselves but still want
  #    the claimant's view reflected in their own can be specified in a
  #    `recognized_by=*` tag.
```

Might add another sentence to the 1. point at the end about this new sub case?